### PR TITLE
Feature/my results

### PR DESCRIPTION
**Pull request by igormarchenko**

### CODE REVIEW

**Comment by sergpet** [comment]
@igormarchenko please rebase

**Comment by igormarchenko** [comment]
@sergpet please merge.

File: Gruntfile.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ module.exports = function (grunt) {
                     'src/main/webapp/plugins/bower/blockUI/jquery.blockUI.js',
                     'src/main/webapp/plugins/bower/angular-validation-match/dist/angular-validation-match.js',
                     'src/main/webapp/plugins/jquery.formstyler/jquery.formstyler.min.js',
+                    'src/main/webapp/plugins/bower/select2/dist/js/select2.full.js',
```
Comment (sergpet) [inline Gruntfile.js:16]: why so ugly naming ? select2 ?

File: src/main/java/org/soippo/controller/UserController.java [hunk 2]
```diff
@@ -109,4 +137,9 @@ public ResponseEntity saveUser(@RequestBody String userData) throws IOException
             return ResponseEntity.badRequest().body(objectMapper.writeValueAsString(ex.getErrorCode()));
         }
     }
+
+    private UserDetails getCurrentUser() {
+        Authentication auth = SecurityContextHolder.getContext().getAuthentication();
```
Comment (sergpet) [inline src/main/java/org/soippo/controller/UserController.java:142]: :+1:

File: Gruntfile.js (hunk 1 above)
Comment (igormarchenko) [inline Gruntfile.js:16]: [plugin name](https://select2.github.io/)

**Pull request merged**